src/mudlet-lua/lua/geyser/GeyserColor.lua

```diff
@@ -148,6 +149,22 @@ function Geyser.Color.parse(red, green, blue, alpha)
   return r, g, b, a
 end
 
+--- Searches for a close match to 'color' in the color_table
```

Comment:
```suggestion
--- Returns for a close match to 'color' in the color_table
```

It's silly but it can be read as 'searches and does what?'

Reply:
added in the second line, will that get picked up? If not, maybe `Searches for and returns a close match to 'color' in the color_table. Returns false if no match found`

Reply:
ugh github, why you no refresh consistently? Disregard, I see the approval now.